openlibrary/tests/core/test_db.py

```diff
@@ -517,6 +520,14 @@ def setup_method(self):
     def teardown_method(self):
         self.db.query('delete from yearly_reading_goals')
 
+    def test_total_yearly_reading_goals(self):
+        assert (
+            BookshelvesEvents.total_yearly_reading_goals() == 3
+        )  # getting <Storage {'count(*)': 3}> instead of 3
+        assert (
+            BookshelvesEvents.total_yearly_reading_goals(since="2022-12-25") == 2
+        )  # Combination of issues
```

Comment:
Please move comments to the preceding line because [assert is not a function in Python](https://peps.python.org/pep-0679/).
```suggestion
        # getting <Storage {'count(*)': 3}> instead of 3
        assert BookshelvesEvents.total_yearly_reading_goals() == 3
        # Combination of issues
        assert BookshelvesEvents.total_yearly_reading_goals(since="2022-12-25") == 2
```